services/cannon/bench/Main.hs

```diff
@@ -0,0 +1,31 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2021 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module Main where
+
+import Bench
+import qualified Cannon.API
+import Data.Metrics.Test (pathsConsistencyCheck)
+import Data.Metrics.WaiRoute (treeToPaths)
+import Imports
+import Network.Wai.Utilities.Server (compile)
+import qualified Test.Cannon.Dict as D
+import Test.Tasty
+import Test.Tasty.HUnit
```

Comment:
Do you need all these imports? The only imported symbol I see is `benchmark`... :thinking:

Reply:
I think `import Bench` should be sufficient.

Reply:
Yes, fixed in aed403d69ca497f01d590ac4efdb036cf0a418d3.